src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/service/reports/MoveFilterer.kt

```diff
@@ -45,7 +45,7 @@ object MoveFilterer {
         count { it.stateIsAnyOf(JourneyState.completed) } == 1 &&
           count { isCompleteBillableJourneyAndLocationsMatchMove(it, move) } == 1 &&
           count { it.stateIsAnyOf(JourneyState.cancelled) } == 0
-      }
+      } && move.hasNoneOf(EventType.MOVE_REDIRECT)
```

Comment:
So for now I have kept the change very simple.  We may need to in the future include a time check on the occurred_at time.